src/index.ts

```diff
@@ -199,11 +201,22 @@ export const createInstance = (config: Config): Instance => {
       amount: ethers.BigNumber,
       signer: ethers.Signer
     ): Promise<ethers.ContractTransaction> => {
-      const registrar: Registrar = await getRegistrar(signer, config.registrar);
+      const hub: ZNSHub = await getHubContract(signer, config.hub);
 
-      const tx = await actions.setDomainRoyalty(
-        domainId,
-        amount,
+      const tx = await actions.setDomainRoyalty(domainId, amount, signer, hub);
+      return tx;
+    },
+    transferOwner: async (
+      to: string,
+      tokenId: string,
+      signer: ethers.Signer
+    ): Promise<ContractTransaction> => {
+      const hub: ZNSHub = await getHubContract(signer, config.hub);
+      const registrar: Registrar = await getRegistrarForDomain(hub, tokenId);
+
+      const tx = await actions.transferOwnership(
```

Comment:
to be consistent with other actions it should take a hub as a parameter not the specific registrar, or it could just take `config.hub` as a parameter and do both calls internally like I mentioned above.